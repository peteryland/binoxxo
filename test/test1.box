x3x5
3o
oo3o2x
1o2x3o
4x2x
9o
1o4ox
7x
1oo3o
2x2o1x
