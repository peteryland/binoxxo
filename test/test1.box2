X___X_____
___O______
OO___O__X_
_O__O___O_
___X__O___
_O__O____O
_O____OX__
_______O__
_OO___O__X
__X__O_X__
