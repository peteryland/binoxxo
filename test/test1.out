X___X_____
___O______
OO___O__X_
_O__X___O_
____X__X__
_________O
_O____OX__
_______X__
_OO___O___
__X__O_X__

XXOOXXOOXO
XXOOXXOXOO
OOXXOOXOXX
XOXOXOXOOX
OXOOXXOXXO
XXOXOOXOXO
OOXXOXOXOX
OXXOXOXXOO
XOOXOXOOXX
OOXXOOXXOX
